src/averageAges.js

```diff
@@ -53,6 +86,26 @@ function calculateWomenAverageAge(people, withChildren) {
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
   // write code here
+
+  const children = people.filter(child => people
+    .some(mother => child.mother === mother.name));
+
+  let averageAge;
+
+  onlyWithSon
+    ? averageAge = children.filter(son => son.sex === `m`)
+      .map(son => son.born - people
+        .find(mother => son.mother === mother.name).born)
+      .reduce((acc, el) => acc + el) / children.filter(son => son.sex === `m`)
+      .map(son => son.born - people
+        .find(mother => son.mother === mother.name).born).length
```

Comment:
```suggestion
      .map(son => son.born - people
      .find(mother => son.mother === mother.name).born)
      .reduce((acc, el) => acc + el) / children.filter(son => son.sex === `m`)
      .map(son => son.born - people
      .find(mother => son.mother === mother.name).born).length
```
все отступы  должны быть одинаковой величины